content/en/boilerplates/notes/1.3.md

```diff
@@ -0,0 +1,75 @@
+
+## Traffic Management
+
+- **Added** the capability for Istio to automatically determine protocol types avoiding the need for specifying the mandatory container ports field in the pod spec.
+- **Added** a mode to the Gateway API for mutual TLS operation that is capable of reading certificates via SDS.
+- **Added** enhancements to the `EnvoyFiltier` API such that all configuration can be added or updated.
+- **Fixed** Sidecar proxy readiness checks now check Envoy's readiness status.
+- **Fixed** issues with protocols like MySQL and MongoDB where the service communicates over the network first in permissive mutual TLS mode.
+- **Improved** Redis load balancer defaults to `MAGLEV` when using the Redis proxy.
+- **Improved** traffic management to direct traffic to the same region and zone.
+- **Improved** Pilot performance.  CPU utilization may be up to 10 times lower.
+- **Improved** the capabilities of `ServiceEntries` to enable the same hostname in different namespaces.
+- **Improved** the capability of the Sidecar API to customize `OutboundTrafficPolicy`.
+
+## Security
+
+- **Added** a unified logging pattern to Citadel agent.
+- **Added** metrics into Citadel agent to monitor SDS push.
+- **Added** the capability to ingress gateway to provide client CA certificates when using SDS.
+- **Added** the capability for trust domain validation when sidecars use mutual TLS.
+- **Added** the labels `ca.istio.io/env` and `ca.istio.io/override`  to allow operators to control service account secret generation by namespace.
+- **Removed** integration with Vault CA temporarily. This was caused by SDS requirements and will be reintroduced in a new release.
+- **Improved** the performance of secrets handling in the sidecars by introducing SDS.
```

Comment:
I don't understand what this is even describing? what was wrong with `Updated SDS feature to be enabled for the Istio control plane.`? It is succinct and describes exactly what changed.

Reply:
My understanding is that every envoy sidecar must mount secrets.  This secret mounting takes upwards of 60-90 seconds.  With SDS, this time is shorter.  Given my understanding of the feature, the above is correct.  @myidpt ?

Reply:
maybe we are thinking about 2 completely different features then. I thought this was about sds support for pilot and galley etc. Oliver can clarify

Reply:
good.

Reply:
already accepted in a prior PR.